### DISCUSSION TITLE
Pulse count

### DISCUSSION THREAD

**Discussion by kimza90** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#101] [General]
hi I'm wondering if I can use CF1 to CF1-2 to count pulses from my energy meter?
 it pulses 1000 times per. kWh. how is this set up then?

**Comment by CircuitSetup** [comment]
You can do this, yes. That is the pulse for active power. This is useful if you need to get data into a system that has a pulse counter. 

If you are looking to get kWh, there are much easier ways to do this via esphome or other software.

**Reply by kimza90** [reply]
I'll already put this in the fuse box. so easiest is to use it if not I have to put more esp32 in the fuse box.
